core/src/main/scala/kafka/server/KafkaRequestHandler.scala

```diff
@@ -192,19 +197,26 @@ class KafkaRequestHandler(
 
 }
 
+object KafkaRequestHandlerPool {
+  val aggregateThreads = new AtomicInteger(0)
+  val requestHandlerAvgIdleMetricName = "RequestHandlerAvgIdlePercent"
```

Comment:
In Scala "constants" at `object` use upper camel case by convention. E.g. `RequestHandlerAvgIdleMetricName`.